yt_dlp/extractor/motherless.py

```diff
@@ -147,102 +146,88 @@ def _real_extract(self, url):
         }
 
 
-class MotherlessGroupIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?motherless\.com/gv?/(?P<id>[a-z0-9_]+)'
+class MotherlessPaginatedIE(InfoExtractor):
+    _PAGE_SIZE = 60
+
+    def _extract_entries(self, webpage, base):
+        for mobj in re.finditer(r'href=".*(?P<href>\/[A-F0-9]+)"\s+title="(?P<title>[^"]+)', webpage):
+            video_url = compat_urlparse.urljoin(base, mobj.group("href"))
+            video_id = MotherlessIE.get_temp_id(video_url)
+
+            if video_id:
+                yield self.url_result(
+                    video_url, ie=MotherlessIE.ie_key(), video_id=video_id,
+                    video_title=mobj.group('title'))
+
+    def _real_extract(self, url):
+        _id = self._match_id(url)
+        webpage = self._download_webpage(url, _id)
+        title = self._search_regex(r'^([\w\s]+)\s+', self._html_extract_title(webpage), 'title')
```

Comment:
Why the search_regex?

Reply:
Title looks like so:

```
<title>F3C5049 | MOTHERLESS.COM ™</title>
```

I could also

```
title = "|".join(self._html.extract_title(webpage).split('|')[:-1]
```

Reply:
Actually also like so:

```html
<title>Movie Scenes - Videos - Hot and sexy scenes from &quot;regular&quot; movies... Beautiful actresses fully | MOTHERLESS.COM ™</title>
```

where `Movie Scenes` would be the title we're looking for, so the regex works.

Reply:
Why is "Movie Scenes" the title here? The whole thing except the suffix should be the title. So do `strip_or_none(remove_end(..., '| MOTHERLESS.COM '))`

Reply:
It is three things:

- Movie scenes (title)
- Video (meaning videos, as opposed to images)
- Hot and sexy scenes .... (description)

But that is handled inconsistently between groups and galleries.